### PR TITLE
Update 2023-02-14.md

### PR DESCRIPTION
**Pull request by danieltprice**
Revise entry
https://neon-next-git-dprice-revise-data-layer-evic-50c332-neondatabase.vercel.app/docs/release-notes

### CODE REVIEW

File: content/release-notes/2023-02-14.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ label: 'Storage'
...
 - Proxy: Significantly reduced network latencies for WebSocket and pooled connections by implementing caching mechanism for compute node connection information and enabling the `TCP_NODELAY` protocol option. The `TCP_NODELAY` option causes segments to be sent as soon as possible, even if there is only a small amount of data. For more information, refer to the [TCP protocol man page](https://linux.die.net/man/7/tcp).
-- Pageserver: Added an experimental feature that enforces a configurable data layer eviction policy. This feature periodically checks when a data layer was last accessed and evicts it if the last-access timestamp is earlier than a defined threshold.
+- Pageserver: Added an experimental feature that enforces a configurable data layer eviction policy. This feature periodically checks when a data layer was last accessed and evicts it if the timestamp of the last access is older than a time threshold defined by the eviction policy. The data required to reconstruct page versions is stored as data layers on Neon Pageservers. Evicting unused data layers helps manage data storage size.
```
Comment (problame) [inline content/release-notes/2023-02-14.md:9]:
```suggestion
- Pageserver: Added an opt-in feature for automatically evicting layer files from Pageservers to optimize local storage space usage. If enabled for a given project, Pageserver periodically checks the access timestamps of the project's layer files. Suppose the most recent access to a layer file is further in the past than a configurable threshold. In that case, Pageserver removes its local copy of the layer file. The authoritative copy remains in S3. It is downloaded on-demand if the layer is needed again, e.g., to reconstruct a page version for a client request.
```

**Comment by danieltprice** [comment]
@problame Thank you for the suggested changes. I revised the wording a bit and kept the "experimental" term to indicate that it's not yet in production.

**Review by problame — DISMISSED** [review]
s/an authoritative copy/the authoritative copy/

otherwise lgtm

**Comment by danieltprice** [comment]
> s/an authoritative copy/the authoritative copy/
> 
> otherwise lgtm

Fixed. TY

**Review by problame — APPROVED** [review]

**Pull request merged**